tests/installation/ipxe_install.pm

```diff
@@ -74,8 +74,14 @@ sub set_bootscript {
     my $ip          = inet_ntoa(inet_aton($host));
     my $http_server = get_required_var('IPXE_HTTPSERVER');
     my $url         = "$http_server/$ip/script.ipxe";
+    my $instparm;
 
-    my $autoyast = get_required_var('AUTOYAST');
+    if (get_var('AUTOYAST')) {
+        my $autoyast = get_var('AUTOYAST');
+        $instparm = "autoyast=$autoyast";
+    } else {
+        $instparm = 'sshd=1 vnc=1 VNCPassword=nots3cr3t sshpassword=nots3cr3t';
```

Comment:
Wouldn't be better to use `$testapi::password` ?

Reply:
oh yes of course